### PR TITLE
vesta: Add version 3.5.7

### PR DESCRIPTION
**Pull request by specter119**
VESTA is a 3D visualization program for structural models, volumetric data such as electron/nuclear densities, and crystal morphologies.

### CODE REVIEW

File: bucket/vesta.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+﻿{
+    "version": "3.5.7",
+    "description": "VESTA is a 3D visualization program for structural models, volumetric data such as electron/nuclear densities, and crystal morphologies.",
```
Comment (Ash258) [inline bucket/vesta.json:3]: Mentioning application name is not needed.

File: bucket/vesta.json [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    "homepage": "https://jp-minerals.org/en/",
+    "license": {
+        "identifier": "Proprietary",
+        "url": "https://jp-minerals.org/vesta/en/download.html"
+    },
+    "url": "https://jp-minerals.org/vesta/archives/3.5.7/VESTA-win64.zip",
```
Comment (Ash258) [inline bucket/vesta.json:9]: ???

File: bucket/vesta.json [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+            "url": "https://jp-minerals.org/vesta/archives/3.5.7/VESTA.zip",
+            "hash": "35c0e7483781398163a5e1472898e4243426691977ca6aee7b4096a41fc10024",
+            "extract_dir": "VESTA"
+        }
+    },
+    "checkver": {
+        "url": "https://jp-minerals.org/vesta/en/download.html",
+        "re": "ver. ([\\d.]+),"
+    },
+    "bin": "VESTA.exe",
```
Comment (Ash258) [inline bucket/vesta.json:26]: What are available command line parameters?

File: bucket/vesta.json [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+        "32bit": {
+            "url": "https://jp-minerals.org/vesta/archives/3.5.7/VESTA.zip",
+            "hash": "35c0e7483781398163a5e1472898e4243426691977ca6aee7b4096a41fc10024",
+            "extract_dir": "VESTA"
+        }
+    },
+    "checkver": {
+        "url": "https://jp-minerals.org/vesta/en/download.html",
+        "re": "ver. ([\\d.]+),"
+    },
```
Comment (Ash258) [inline bucket/vesta.json:25]:
```suggestion
```

File: bucket/vesta.json [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    "bin": "VESTA.exe",
+    "shortcuts": [
+        [
+            "VESTA.exe",
+            "VESTA"
+        ]
+    ],
```
Comment (Ash258) [inline bucket/vesta.json:32]:
```suggestion
    ],
    "checkver": {
        "url": "https://jp-minerals.org/vesta/en/download.html",
        "regex": "\\(ver\\.\\s+([\\d.]+),"
    },
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/vesta.json (hunk 3 above)
Comment (specter119) [reply bucket/vesta.json]: Seems command is useless, drop it.

File: bucket/vesta.json (hunk 2 above)
Comment (specter119) [reply bucket/vesta.json]: this url removed

File: bucket/vesta.json (hunk 1 above)
Comment (specter119) [reply bucket/vesta.json]: description updated

**Comment by specter119** [comment]
@Ash258 all check have been passed

File: bucket/vesta.json [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    },
+    "shortcuts": [
+        [
+            "VESTA.exe",
+            "VESTA"
+        ]
+    ],
+    "checkver": {
+        "url": "https://jp-minerals.org/vesta/en/download.html",
+        "re": "ver. ([\\d.]+),"
```
Comment (Ash258) [inline bucket/vesta.json:29]:
```suggestion
        "regex": "ver\\.\\s+([\\d.]+),"
```

**Review by Ash258 — CHANGES_REQUESTED** [review]
- What are all the ini files in application directory?
- Where do application store settings?

File: bucket/vesta.json (hunk 6 above)
Comment (specter119) [reply bucket/vesta.json]: persist style.ini for vesta

**Review by Ash258 — CHANGES_REQUESTED** [review]
- What are **all the ini** files in application directory?
- Where do application store settings?

**Pull request closed without merging**